cypress/e2e/settings/users/patron-block/maximum.charged.out.returned.cy.js

```diff
@@ -0,0 +1,204 @@
+          permissions.checkinAll.gui,
+          permissions.checkoutAll.gui,
+          permissions.uiUsersView.gui,
+        ],
+        patronGroup.name
+      )
+        .then((userProperties) => {
+          userData.username = userProperties.username;
+          userData.password = userProperties.password;
+          userData.userId = userProperties.userId;
+          userData.barcode = userProperties.barcode;
+          userData.personal.lastname = userProperties.lastName;
+        })
+        .then(() => {
+          UserEdit.addServicePointViaApi(testData.userServicePoint.id, userData.userId, testData.userServicePoint.id);
+
+          cy.get('@items').each((item) => {
+            Checkout.checkoutItemViaApi({
+              id: uuid(),
+              itemBarcode: item.barcode,
+              loanDate: moment.utc().format(),
+              servicePointId: testData.userServicePoint.id,
+              userBarcode: userData.barcode,
+            });
+          });
+
+          cy.login(userData.username, userData.password);
+        });
+    });
+  });
+
+  after('Deleting created entities', () => {
+    cy.get('@items').each((item) => {
+      CheckInActions.checkinItemViaApi({
+        itemBarcode: item.barcode,
+        servicePointId: testData.userServicePoint.id,
+        checkInDate: new Date().toISOString(),
+      });
+    });
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    Conditions.resetConditionViaApi('3d7c52dc-c732-4223-8bf8-e5917801386f', 'Maximum number of items charged out');
+  });
+
+  it(
+    'C350647 Verify automated patron block "Maximum number of items charged out" removed after charged item returned (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      cy.visit(SettingsMenu.conditionsPath);
+      Conditions.waitLoading();
+      Conditions.select('Maximum number of items charged out');
+      Conditions.setConditionState('Patron is Blocked');
+      cy.visit(SettingsMenu.limitsPath);
+      Limits.selectGroup(patronGroup.name);
+      Limits.setMaximumNumberOfItemsChargedout('4');
+
+      cy.visit(TopMenu.usersPath);
+      UsersSearchPane.searchByKeywords(userData.barcode);
+      Users.checkIsPatronBlocked('Patron is Blocked', 'Borrowing, Renewals, Requests');
```

Comment:
pass the first argument as the variable checkedOutBlockMessage

Reply:
Done